extensions/kafka-streams/deployment/src/main/java/io/quarkus/kafka/streams/deployment/KafkaStreamsProcessor.java

```diff
@@ -196,4 +202,34 @@ void configureAndLoadRocksDb(KafkaStreamsRecorder recorder, KafkaStreamsRuntimeC
     AdditionalBeanBuildItem registerBean() {
         return AdditionalBeanBuildItem.unremovableOf(KafkaStreamsTopologyManager.class);
     }
+
+    @BuildStep
+    void addHealthChecks(KafkaStreamsBuildTimeConfig buildTimeConfig) {
```

Comment:
You can inject the BuillProducer inside your method, this is how it is usually done

```
 void addHealthChecks(KafkaStreamsBuildTimeConfig buildTimeConfig, BuildProducer<HealthBuildItem> healthChecks)
```